microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -1,150 +1,280 @@
+<!-- from paragraph 3 of post from github issue 341: -->
+### Experimental APIs are not guaranteed to be forward-compatible
 
-## Fixed Version distribution mode   
+Any experimental APIs in a pre-release SDK are not guaranteed to be forward-compatible.   When a _pre-release_ version of the WebView2 SDK is initially made available, that SDK only works with Microsoft Edge Canary.  After that, the SDK works with the Beta and Dev channels, for the development phase of your app.  A pre-release version of the WebView2 SDK is meant for developers to try out new APIs early and provide feedback before the new APIs are promoted to become stable, forward-compatible APIs.
 
-For constrained environments with strict compatibility requirements, consider using the Fixed Version distribution mode.  Choose and package a specific version of the WebView2 Runtime using the Fixed Version distribution mode.  You may specify the timing of Runtime updates for your app.  The Fixed Version distribution mode does not receive any automatic updates. Plan to update your app and the Runtime.  
+For a release version of your app, use a _Release_ version of the WebView2 SDK.  The _stable_ (non-experimental) APIs that are in that SDK version are forward-compatible.  
 
-> [!NOTE] 
-> The Fixed Version distribution mode was previously called bring-your-own.  
+The WebView2 SDK has been forward-compatible ever since version 1.  If your WebView2 app uses the current version of the SDK, your app continues to work even after the client's WebView2 Runtime is updated.  However, any experimental APIs that are in a version of the WebView2 SDK are not guaranteed to be forward-compatible.
+<!-- /para3 -->
 
-To use the Fixed Version mode, complete the following actions
 
-1.  [Download][Webview2Installer] the Fixed Version package. 
-1.  Decompress the package using command line `expand {path to the package} -F:* {path to the destination folder}` or with tools such as WinRAR. Avoid decompressing through the File Explorer as it may not generate the correct folder structure.  
+<!-- ====================================================================== -->
+## Fixed Version distribution mode
+
+For constrained environments with strict compatibility requirements, consider using the Fixed Version distribution mode.  The Fixed Version distribution mode was previously called _bring-your-own_.
+
+To use the Fixed Version approach, get the WebView2 Runtime from the NuGet SDK versions website or from [Download the WebView2 Runtime][Webview2Installer], and then package the Runtime with your WebView2 app.
```

Comment:
```suggestion
To use the Fixed Version approach, get the WebView2 Runtime from [Download the WebView2 Runtime][Webview2Installer], and then package the Runtime with your WebView2 app.
```
Probably an error, delete mention of SDK download site, now that SDK & Runtime are differentiated.